src/api/java/baritone/api/Settings.java

```diff
@@ -847,6 +847,11 @@
      */
     public final Setting<Boolean> skipFailedLayers = new Setting<>(false);
 
+    /**
+     * Only build the selected part of the schematic when using #schematica
+     */
+     public final Setting<Boolean> schematicaOnlyBuildSelection = new Setting<>(false);
```

Comment:
Maybe something like buildOnlySelection would be better.

Reply:
Sure, I will change it tomorrow.